src/store/builder/f0492216-65f3-4dc7-a1d8-8f79d1d30c11-flow.json

```diff
@@ -100,6 +100,49 @@
               "max_digits": -1
             }
           }
+        },
+        {
+          "uuid": "3f01148c-0945-4f2f-808e-15039cbd962c",
+          "ui_metadata": {
+            "canvas_coordinates": {
+              "x": 888,
+              "y": 612
+            }
+          },
+          "vendor_metadata": {
+            "io_viamo": {
+              "uiData": {
+                "isSelected": false
+              },
+              "branchingType": "UNIFIED",
+              "noValidResponse": "END_CALL"
+            }
+          },
+          "type": "Core.SetGroupMembership",
+          "name": "abc",
+          "label": "abc",
+          "semantic_label": "",
+          "config": {
+            "group_key": "",
+            "group_name": "",
+            "is_member": null
+          },
+          "exits": [
+            {
+              "uuid": "6c9aee4f-afce-484c-8652-356777bf686b",
+              "name": "1",
+              "test": "block.value = true",
+              "config": {}
+            },
+            {
+              "uuid": "9c7c4aa0-14ed-482f-b56d-02ebb0ada9c1",
+              "name": "Default",
+              "default": true,
+              "test": "",
+              "config": {}
+            }
+          ],
+          "tags": []
```

Comment:
What is the purpose of these json files? Are they for testing?

Reply:
Yes, they're test files. Adding this block is intended to show that you can import a tree with blocks that aren't totally valid (`is_member` is null here). The intention is that the UI for fixing this kind of thing is well developed in the InteractionDesigner.

Reply:
Okay. Do you mean unit tests? Or manual tests? In any case, how can I try and see it in action?

Reply:
This is for manual testing. You can try it on the import page and see how it renders in the Interaction Designer.

Reply:
Ah, I see now. I just had to use the `Paste JSON directly` option.
I thought that maybe the flow-builder was set up to read it automatically from the `src/store/builder` folder (along with other files) when in the dev mode 🙂

Reply:
You can also upload them - either way should work the same.